scripts/tests/test_validate_docstrings.py

```diff
@@ -1052,6 +1052,11 @@ def test_raises_for_invalid_attribute_name(self, invalid_name):
         with pytest.raises(AttributeError, match=msg):
             validate_docstrings.Docstring(invalid_name)
 
+    @pytest.mark.parametrize('name', ['pandas.Series.str.isdecimal'])
```

Comment:
If only using one value no need to parametrize - just use this locally in function

Reply:
In the end I added a second test case to check.